src/components/Frame/Frame.scss

```diff
@@ -29,11 +34,16 @@ $skip-vertical-offset: rem(10px);
 
   @include frame-when-nav-displayed {
     z-index: 1;
+    // `calc()` needs the px value below.
+    // stylelint-disable-next-line length-zero-no-unit
+    left: calc(0px + var(--p-frame-offset));
```

Comment:
Why does this one need to be in a calc at all? Couldn’t we just use the var here?

Reply:
Um, yes that should be fine

I must have set this up before defaulting to 0px in the util 👍